pkg/dedicatedadmin/dedicatedadmin.go

```diff
@@ -44,15 +44,13 @@ func IsBlackListedNamespace(namespace string, blacklistedNamespaces string) bool
 
 // GetOperatorConfig gets the operator's configuration from a config map
```

Comment:
```suggestion
// GetOperatorConfig gets the operator's configuration from a config map
// TODO: Refactor this as per issue #16
```

Added a note around this in light of #16. Surely could have this note peppered all around the code, but here (at least) makes the most sense.